code/modules/antagonists/hivemind/vessel.dm

```diff
@@ -41,6 +41,7 @@
 	..()
 
 /datum/antagonist/hivevessel/on_removal()
+	remove_innate_effects()
```

Comment:
This normally gets called during the base `/datum/antagonist/on_removal`, but we set `master` to null before we call `..()`.

We also can't just run `..()` first, as `/datum/antagonist/on_removal` will qdel src at the end!